### PR TITLE
meson: add support for onecloud

### PR DESCRIPTION
**Pull request by shiyu1314**
Specifications:

SoC: Amlogic S805 ARM Cortex-A53 1.5 GHz
RAM: 1 GB RAM
Flash: 8 GB eMMC
LAN: 1x RTL8211F Gigabit Ethernet/at support
LEDs: 1x blue and 1x red
USB：Two USB 2.0
Buttons: 1x reset button

Flashing instructions:
  Standard onecloud SD card installation procedure - copy image to SD card,
  insert into SD card slot on the device and boot.

It can also be made into a direct brush bag
https://github.com/shiyu1314/u-boot-onecloud/releases

### CODE REVIEW

File: target/linux/meson/Makefile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+ARCH:=arm
+BOARD:=meson
+BOARDNAME:=Amlogic Meson family
+FEATURES:=audio boot-part ext4 fpu legacy-sdcard squashfs usbgadget
+MAINTAINER:=shiyu1314 <1052672227@qq.com>
+SUBTARGETS:=meson8b
+
+KERNEL_PATCHVER:=5.15
+KERNEL_TESTING_PATCHVER:=6.1
```
Comment (Ansuel) [inline target/linux/meson/Makefile:15]: I would directly jump for 6.1. Is it problematic? Do we need 5.15 for any reason?

Comment (shiyu1314) [reply target/linux/meson/Makefile:15]: No problem, one more choice

Comment (shiyu1314) [reply target/linux/meson/Makefile:15]: No problem, one more choice

Comment (shiyu1314) [reply target/linux/meson/Makefile:15]: > I would directly jump for 6.1. Is it problematic? Do we need 5.15 for any reason?

No problem, one more choice

**Comment by motolav** [comment]
@shiyu1314 stop closing PRs for changes, force push changes to update your PR

**Comment by motolav** [comment]
Follow the guide here instead of opening new PRs
https://openwrt.org/docs/guide-developer/working-with-github-pr

**Pull request closed without merging**